README.md

```diff
@@ -111,7 +111,7 @@ Then, add this plugin:
               <dependency>
                   <groupId>org.cqfn.diktat</groupId>
                   <artifactId>diktat-rules</artifactId>
-                  <version>1.0.1</version> <!-- replace it with diktat latest version -->
+                  <version>1.0.2-SNAPSHOT</version> <!-- replace it with diktat latest version -->
```

Comment:
The released version is `1.0.2`, the `snapshot` suffix is for development phase only.

Reply:
fixed